gamemodes/cinema/gamemode/modules/theater/services/sh_dlive.lua

```diff
@@ -0,0 +1,82 @@
+-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+
+local SERVICE = {}
+
+SERVICE.Name 		= "DLive"
```

Comment:
Why the hell is there a large gap between SERVICE.Name and the equal sign?

Reply:
it was originally so that the equals signs line up vertically between different member variables (some with longer names), if that's not the case here it was just copied